crates/relay/src/lib.rs

```diff
@@ -106,7 +105,7 @@ pub mod pallet {
     #[pallet::storage]
     #[pallet::getter(fn theft_report)]
     pub(super) type TheftReports<T: Config> =
-        StorageMap<_, Blake2_128Concat, H256Le, BTreeSet<T::AccountId>, ValueQuery>;
+        StorageDoubleMap<_, Blake2_128Concat, T::AccountId, Blake2_128Concat, H256Le, Option<()>, ValueQuery>;
```

Comment:
Is there any reason to store the txid at all? If we include the txid in the liquidation status as discussed on discord, that is. 

Also, I think it would be slightly cleaner to store `()`, and to make it an OptionQuery

Reply:
Also maybe add a comment that we're using this storage as a set rather than a map